### PR TITLE
dynamic navbar

### LINKED ISSUE #5: Dynamically create navbar from route data
The following code pulls data from `data/routes.yml` and lets you loop over each element.

In the future we can add a layouts feature to the navbar yaml so that only navbar pages with exactly four routes are told to use @EliasJRH's cool logo looking nav layout. For now, just make the navbar generate with the template he has already created.

```
<ul>
{{ range $.Site.Data.routes }}
    <li>{{ .url }}</li>
    <li>
        <ul>
        {{ range .sublinks }}
            <li>{{ .url }}</li>
            <li><img src='{{ absURL "" }}{{ .image }}' width="100px"/></li>
        {{ end }}
        </ul>
    </li>
{{ end }}
</ul>
```

### PR DESCRIPTION
**Pull request by EliasJRH**
closes #5

### CODE REVIEW

File: assets/scss/components/_navbar.scss [hunk 1]
```diff
@@ -104,11 +104,12 @@ li {
 
 #item1,
 #item2,
-#item3 {
+#item3,
+#item4 {
```
Comment (MathyouMB) [inline assets/scss/components/_navbar.scss]: you could add this style to the item class

Comment (MathyouMB) [reply assets/scss/components/_navbar.scss]: actually I believe this is a duplicate of what you do on line 141

Comment (EliasJRH) [reply assets/scss/components/_navbar.scss]: ah yea, I see it

Comment (EliasJRH) [reply assets/scss/components/_navbar.scss]: should be cleaner now

**Review by MathyouMB — APPROVED** [review]
LGTM! Good job on the css iterator. 👍

**Pull request merged**